redhat-developer/developers.redhat.com
# [SURGE 3106] Collections assembly

### JIRA Issue Link
* https://github.com/redhat-developer/developers.redhat.com/issues/3106

### Verification Process
1. view `/training`
2. Check Collections assembly matches designs
